[2022-06-15 16:23:05,305] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task manual__2022-06-15T16:23:02.637168+00:00 [queued]>
[2022-06-15 16:23:05,318] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task manual__2022-06-15T16:23:02.637168+00:00 [queued]>
[2022-06-15 16:23:05,321] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-15 16:23:05,322] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2022-06-15 16:23:05,323] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-15 16:23:05,340] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): second_task> on 2022-06-15 16:23:02.637168+00:00
[2022-06-15 16:23:05,346] {standard_task_runner.py:52} INFO - Started process 994 to run task
[2022-06-15 16:23:05,350] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'second_task', 'manual__2022-06-15T16:23:02.637168+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp8r5rmpqx', '--error-file', '/tmp/tmp4wimqcax']
[2022-06-15 16:23:05,353] {standard_task_runner.py:80} INFO - Job 7: Subtask second_task
[2022-06-15 16:23:05,425] {task_command.py:370} INFO - Running <TaskInstance: our_first_dag_v2.second_task manual__2022-06-15T16:23:02.637168+00:00 [running]> on host e2977833d61b
[2022-06-15 16:23:05,516] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=phersonj
AIRFLOW_CTX_DAG_ID=our_first_dag_v2
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-15T16:23:02.637168+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-15T16:23:02.637168+00:00
[2022-06-15 16:23:05,518] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-15 16:23:05,520] {subprocess.py:74} INFO - Running command: ['bash', '-c', "echo hey, I'm the second task and I'll be running after task 1"]
[2022-06-15 16:23:05,532] {subprocess.py:85} INFO - Output:
[2022-06-15 16:23:05,535] {subprocess.py:92} INFO - hey, Im the second task and Ill be running after task 1
[2022-06-15 16:23:05,536] {subprocess.py:96} INFO - Command exited with return code 0
[2022-06-15 16:23:05,576] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, execution_date=20220615T162302, start_date=20220615T162305, end_date=20220615T162305
[2022-06-15 16:23:05,604] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-15 16:23:05,648] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
